### PR TITLE
Fix some upgrade logic

### PR DESCRIPTION
**Pull request by stephencelis**
Coupla small gotchas.

### CODE REVIEW

File: Sources/PointFree/Account/Change.swift [hunk 1]
```diff
@@ -56,12 +56,20 @@ private func subscriptionChange(_ conn: Conn<StatusLineOpen, (Stripe.Subscriptio
 
     let (currentSubscription, _, _, newPricing) = conn.data
 
-    let newPrice = defaultPricing(for: newPricing.lane, billing: newPricing.billing) * 100
-    let currentPrice = currentSubscription.plan.amount.rawValue
+    let newPrice = (defaultPricing(for: newPricing.lane, billing: newPricing.billing) * 100) * newPricing.quantity
```
Comment (stephencelis) [inline Sources/PointFree/Account/Change.swift]: Weren't considering quantity at all in proration 🤦‍♂️

File: Sources/PointFree/Account/Change.swift [hunk 2]
```diff
@@ -56,12 +56,20 @@ private func subscriptionChange(_ conn: Conn<StatusLineOpen, (Stripe.Subscriptio
...
+    let currentPrice = currentSubscription.plan.amount.rawValue * currentSubscription.quantity
 
     let prorate = newPrice > currentPrice
 
     return AppEnvironment.current.stripe.updateSubscription(currentSubscription, newPricing.plan, newPricing.quantity, prorate)
+      .flatMap { sub -> EitherIO<Error, Stripe.Subscription> in
+        if prorate {
+          parallel(AppEnvironment.current.stripe.invoiceCustomer(sub.customer).run)
```
Comment (stephencelis) [inline Sources/PointFree/Account/Change.swift]: I'm pretty sure I was seein automatic proration when omitting the flag entirely, but in testing was seeing that it wouldn't invoice when explicit.

File: Sources/PointFree/Pricing.swift [hunk 3]
```diff
@@ -335,7 +335,7 @@ private let pricingIntervalRowView = View<Pricing> { pricing in
               Class.type.normal,
               ])
             ],
-            ["Up to 20% off the Individual Monthly plan"])
+            ["20% off the Individual Monthly plan"])
```
Comment (stephencelis) [inline Sources/PointFree/Pricing.swift]: Lil copy change. It's actually more like 22% off :P

Comment (mbrandonw) [reply Sources/PointFree/Pricing.swift]: BIG SAVINGS

File: Sources/PointFree/Account/Change.swift (hunk 2 above)
Comment (mbrandonw) [reply Sources/PointFree/Account/Change.swift]: So this is a fire and forget invoice? What happens if it fails? Does that matter?

Comment (stephencelis) [reply Sources/PointFree/Account/Change.swift]: Ah i can do an email

**Pull request merged**